src/corehost/cli/fxr/hostfxr.cpp

```diff
@@ -87,6 +87,7 @@ bool hostpolicy_exists_in_svc(pal::string_t* resolved_dir)
 {
     pal::string_t svc_dir;
     pal::get_default_extensions_directory(&svc_dir);
+    append_path(&svc_dir, _X("msil"));
```

Comment:
I think we should call the subfolder containing non-R2R packages as "pkgs", and not "msil", since it will contain packages that are not just MSIL (e.g. CoreCLR).

Please fix the other location where "msil" was committed.

CC @leecow @ericstj